lib/_debugger.js

```diff
@@ -1690,8 +1690,8 @@ Interface.prototype.trySpawn = function(cb) {
     // pipe stream into debugger
     this.child = spawn(process.execPath, childArgs);
 
-    this.child.stdout.on('data', this.childPrint.bind(this));
-    this.child.stderr.on('data', this.childPrint.bind(this));
+    this.child.stdout.on('data', () => this.childPrint());
+    this.child.stderr.on('data', () => this.childPrint());
```

Comment:
Needs to pass `text` https://github.com/nodejs/node/blob/master/lib/_debugger.js#L888